ingestion_tools/scripts/importers/neuroglancer.py

```diff
@@ -78,5 +82,5 @@ def get_shader_controller(cls, tomo_header: np.recarray) -> dict[str, Any]:
         }
 
     @classmethod
-    def find_ng(cls, config: DataImportConfig, tomo: TomogramImporter) -> list["NeuroglancerImporter"]:
+    def find_ng(cls, config: DepositionImportConfig, tomo: TomogramImporter) -> list["NeuroglancerImporter"]:
         return [cls(config=config, parent=tomo)]
```

Comment:
Is this finder still needed?

Reply:
Yep, there's code in `config.py` that adds neuroglancer and key_images sections to all configs behind the scenes, so the key image and ng importers are definitely still needed!